Library/Formula/gpa.rb

```diff
@@ -0,0 +1,21 @@
+class Gpa < Formula
+  desc "Graphical user interface for the GnuPG"
+  homepage "https://www.gnupg.org/related_software/gpa/"
+  url "ftp://ftp.gnupg.org/gcrypt/gpa/gpa-0.9.9.tar.bz2"
```

Comment:
Hi DomT4, there is no https Version of this download on gnupg.org. There is only this ftp version.